packages/ses/demos/challenge/index.html

```diff
@@ -34,21 +34,16 @@ <h1>SES Challenge</h1>
       the least time, concluding that the full password must start with that
       character. Then they iterate on the second character, and so on until they've
     worked out the full password, roughly 18 seconds later.</p>
-    <p>However an SES-confined program does not get access to non-determinism
-      (except as mediated by the code that built the environment), so this attacker
+    <p>However an SES-confined program only gets to access non-determinism in the start
```

Comment:
```suggestion
    <p>However a SES-confined program only gets to access non-determinism in the start
```
Use your judgement. If people pronounce it "Ess Eee Ess", then "an" is correct.